docs/docs/use-cases/keda.md

```diff
@@ -0,0 +1,201 @@
+---
+comments: true
+---
+
+# Scaling Workloads with KEDA based on Keptn metrics
+
+If you want to use [KEDA](https://keda.sh/) to scale your workloads, you can set it up
+to consume metrics from Keptn.
+This gives you great flexibility for your setup since Keptn can
+consolidate different observability providers for you and KEDA
+will simplify the scaling operations for you.
+
+This use case is enabled by the Keptn Metrics
+[custom API](https://kubernetes.io/docs/reference/external-api/custom-metrics.v1beta2/)
```

Comment:
Don't you mean the Kubernetes Custom Metrics -- that's where the link goes.  And shouldn't the text for the link match the title of the target page?

Reply:
done